crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1954,6 +1968,11 @@ impl<'db> KnownInstanceType<'db> {
                 .default_ty(db)
                 .map(|ty| ty.to_meta_type(db))
                 .unwrap_or_else(|| KnownClass::NoDefaultType.to_instance(db)),
+            (Self::TypeAliasType(alias), "__name__") => Type::string_literal(db, alias.name(db)),
+            (Self::TypeAliasType(alias), "__value__") => alias.value_ty(db),
+            (Self::TypeAliasType(_), "__type_params__") => {
+                todo_type!("TypeAliasType __type_params__")
+            }
```

Comment:
I'm also not sure we need a match arm and a Todo here, when it would be correct (and match the behavior of other type checkers) to just allow it to fall back to the typeshed definition of the attribute. We can potentially add more precise per-instance typing for it in the future if there are compelling use cases, but we don't need to consider it a todo.